schemafy_lib/src/lib.rs

```diff
@@ -327,7 +353,8 @@ impl<'r> Expander<'r> {
         };
         let s = &s.to_pascal_case();
         let s = replace_invalid_identifier_chars(s);
-        replace_numeric_start(&s)
+        replace_numeric_start(&s);
```

Comment:
```suggestion
        let s = replace_numeric_start(&s);
```

? And add a test to catch this perhaps

Reply:
@Marwes , thank you for your review! Good catch :)

Actually, removing underscores is unnecessary here, because `type_ref` uses `PascalCase`, so no underscores are expected.

However, I did write a small test for the function and found a potential problem. The `$ref` is supposed to be a valid `URI` with a "fragment" pointing to a specific object in a file, so just taking everything after the last `/` won't work in some cases.

I tried to implement the "proper" URI parsing with some fallback mechanisms. Please let me know what you think.